webpack_config/config.js

```diff
@@ -4,7 +4,9 @@ const path = require('path');
 module.exports = {
   port: process.env.HTTPS ? 3443 : 3000,
   title: 'MEW',
-  publicPath: process.env.BUILD_GH_PAGES ? '/react-semantic.ui-starter/' : '/',
+  publicPath: process.env.BUILD_GH_PAGES
```

Comment:
We may as well get rid of the `/react-semantic.ui-starter/` from the original boilerplate, and fix the nested ternary.